### PR TITLE
Pull Request: Submit vignette for Feature computations 20240916

### PR DESCRIPTION
**Pull request by andreabr-dixie**
Pull Request: Submit vignette for Feature computations 20240916

Description
This pull requests that we add a new vignette to utDataStoR - computations. This vignette describes each of the computations in the utMetrics package organized by the type of computation (count, rate, sum) and describes how the year works for each computation category. The utMetrics package is private, so placing this vignette in utDataStoR allows it to be placed in a public bookdown.

Changes Made
Added a new vignette titled "computations"
Includes the following:
A description of each computation organized by the type of computation (count, rate, sum)
A description of how the year works for each computation category.

Motivation and Context
The goal of this vignette is to describe how each computation is calculated and to describe how the year works for each computation category.

How Has This Been Tested?
The vignette has been reviewed for clarity and accuracy by the Office of Institutional Effectiveness.
All code examples have been tested to ensure they produce the expected results.

Styling
The styling of this vignette has been reviewed and approved by the Office of Institutional Effectiveness.

Types of Changes

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
Checklist

 - [x] My vignette follows the style guidelines provided in the README for this repository
 - [x] I have performed a self-review of my own code, including knitting the vignette
 - [x] I have commented my code, particularly in hard-to-understand areas
 - [x] I have reviewed my .gitignore file to ensure the exclusion of items we do not want included in a commit history
 - [x] My vignette does not include any sensitive data

Additional Notes

### CODE REVIEW

**Comment by andreabr-dixie** [comment]
@MattNickodemus please review this vignette today as discussed in morning huddle.

File: vignettes/computations.Rmd [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+- Returned 4th Fall - Student was enrolled at Fall Census and returned in the fall 3 years later (fall 4). For example enrolled in Fall 2020, returned Fall 2023. This is used only for cohort retention.
+
+#### Year
+Data is reported by season (Fall, Spring, Summer) and Year. Season and Year together is referred to as Term. The year in the table and graph for retention metrics is the cohort year tied to the calendar year of the cohort term code (i.e. cohort term code - 202440, Fall 2024, is year 2024) or is the term year of the term being measured. The year is not the reporting year. Each retention metric has the 5 most recent years of data if it is available. If a retention metric needs 3 years to have complete data (returned 4th fall), then the metric will show the most recent complete data (Cohort retention returned 4th fall for the 2023 cohort will not be provided until Fall 2026). 
+
+### 150% Graduation Rate
+
+150% Graduation Rate tracks a cohort of students, after removing exclusions, for 6 years to see if the student earns a credential within 150% of the degree time (150% for a bachelor degree is 6 years) in a manner similar to IPEDS Graduation Rate.
```
Comment (MattNickodemus) [inline vignettes/computations.Rmd:69]: We should add a discussion of how this impacts non bachelors students. We can catch that in the next round of reviews, but I think we'll need something.

**Review by MattNickodemus — APPROVED** [review]

**Comment by MattNickodemus** [comment]
@cdemkegit This is ready to be merged whenever you're ready.

**Comment by MattNickodemus** [comment]
@cdemkegit This on is ready to go as well.

**Comment by cdemkegit** [comment]
I'm merging this to dev

File: vignettes/computations.Rmd (hunk 1 above)
Comment (andreabr-dixie) [reply vignettes/computations.Rmd:69]: I have noted this an made a new issue. #36

**Pull request merged**